README.md

```diff
@@ -7,10 +7,6 @@
 
 Manages staging directory, along with download/extraction of compressed files.
 
-WARNING: Version 0.2.0 no longer uses hiera functions. The same behavior should be available in Puppet 3.0.
-
-NOTE: Version 1.0.0 will be the last feature release. New functionality such as checksum will be implemented in a type/provider module [puppet-archive](https://www.github.com/voxpupuli/puppet-archive).
```

Comment:
In light of there being a 2.0.0 (ruby>1.8) release, this does need updating, but we probably still want to point users towards puppet-archive?